src/Lib/Managers/ItemsManager.vala

```diff
@@ -371,4 +354,140 @@ public class Akira.Lib.Managers.ItemsManager : Object {
 
         window.event_bus.z_selected_changed ();
     }
+
+    /*
+     * Create an item loaded from an opened file.
+     *
+     * @param Json.Object obj - The json object containing the item to load.
+     */
+    public void load_item (Json.Object obj) {
+        Models.CanvasItem? item = null;
+        Models.CanvasArtboard? artboard = null;
+
+        var transform = obj.get_member ("transform").get_object ();
+        var pos_x = transform.get_double_member ("x0");
+        var pos_y = transform.get_double_member ("y0");
+
+        // If item is inside an artboard update the coordinates accordingly.
+        if (obj.has_member ("artboard")) {
+            foreach (var _artboard in artboards) {
+                if (_artboard.id == obj.get_string_member ("artboard")) {
+                    var matrix = Cairo.Matrix.identity ();
+                    _artboard.get_transform (out matrix);
+                    pos_x = matrix.x0 + obj.get_double_member ("initial-relative-x");
+                    pos_y = matrix.y0 + obj.get_double_member ("initial-relative-y");
+                    artboard = _artboard;
+                    break;
+                }
+            }
+        }
+
+        switch (obj.get_string_member ("type")) {
+            case "AkiraLibModelsCanvasRect":
+                insert_type = Models.CanvasItemType.RECT;
+                item = insert_item (pos_x, pos_y);
+                break;
+
+            case "AkiraLibModelsCanvasEllipse":
+                insert_type = Models.CanvasItemType.ELLIPSE;
+                item = insert_item (pos_x, pos_y);
+                break;
+
+            case "AkiraLibModelsCanvasText":
+                insert_type = Models.CanvasItemType.TEXT;
+                item = insert_item (pos_x, pos_y);
+                break;
+
+            case "AkiraLibModelsCanvasArtboard":
+                insert_type = Models.CanvasItemType.ARTBOARD;
+                item = insert_item (pos_x, pos_y);
+                break;
```

Comment:
Here I can see the `CanvasImage` item's loading is not yet implemented.

I don't know if this is due to the fact the we have a separated PR for images and that would have caused merge issue, so I state this here as a reminder!

Reply:
Yeah, I left the image stuff outside as I'm not yet saving images at all.
I will implement this in a separate PR.